include/boost/numeric/odeint/stepper/detail/pid_step_adjuster_coefficients.hpp

```diff
@@ -0,0 +1,166 @@
+#ifndef PID_STEP_ADJUSTER_COEFFICIENTS_HPP_INCLUDED
+#define PID_STEP_ADJUSTER_COEFFICIENTS_HPP_INCLUDED
+
+#include <boost/array.hpp>
+
+namespace boost{
+namespace numeric{
+namespace odeint{
+namespace detail{
+
+enum adjuster_type{
+	BASIC,
+	H0211,
+	H211b,
+	H211PI,
+	H0312,
+	H312b,
+	H312PID,
+	H0321,
+	H321
+};
+
+template<int Type>
+class pid_step_adjuster_coefficients;
+
+template<>
+class pid_step_adjuster_coefficients<BASIC> : public boost::array<double, 5>
```

Comment:
the floating point type should be passed on from the stepper's value_type. But I think this can be done in a later PR, this is quite big already.